Header header
uint8 camera_type
int32 frame_number
vision/MsgBin[] bins
